src/PlatformSockets.c

```diff
@@ -14,18 +14,26 @@
 
 #if defined(LC_WINDOWS)
 
+# if defined(WINAPI_FAMILY) && WINAPI_FAMILY==WINAPI_FAMILY_APP
+# define LC_UWP
+# else 
+# define LC_WINDOWS_DESKTOP
+#endif
```

Comment:
Can you move this into Platform.h where `LC_WINDOWS` is also defined? We only use it here today, but I could see us using it other places eventually too.